src/include/rimage/manifest.h

```diff
@@ -127,6 +128,15 @@ struct fw_image_manifest_v2_5 {
 	struct sof_man_fw_desc desc;	/* at offset MAN_DESC_OFFSET */
 } __attribute__((packed));
 
+struct fw_image_manifest_v2_6 {
+	struct fw_image_manifest_v2_5 man_v2_5;
+	struct fw_ext_manifest ext_man;
+	int mod_cfg_count;
+	struct sof_man_mod_config *mod_cfg;
+	int mod_man_count;
+	struct sof_man_module *mod_man;
+};
```

Comment:
This looks very similar to v2.5 ? Can we reuse ? and set the new 2.6 fields to 0 in 2.5 ?

Reply:
yes, please check my new PR